### ISSUE TITLE
Documentation update on Handling Ontap ls-mirror update for trident volumes

### ISSUE THREAD

**Issue by balahari18** [NetAppDocs/trident#140]
Page: [Frequently asked questions](https://docs.netapp.com/us-en/trident/faq.html)
If LS Mirror is configured for SVM's root volume, is there any recommendation for provisioning PVC?
Ontap automatically updates ls-mirror for trident provisioned volumes. When multiple PVC/PV is created together, We need to ensure Previous Ls-mirror update is successful at ontap and provision the next PVC/PV. If not, the new pvc cannot be mounted until ontap does next ls-mirror update.

Ref - https://jira.ngage.netapp.com/browse/CDS-4472?focusedCommentId=2369069&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2369069

**Comment by juliantap** [comment]
Hi  @balahari18, thank you for submitting this. Looking at Jira, the noted version is 21.07. Can you or @balaramesh or @gnarl confirm this is for 21.07 and later?

**Comment by balaramesh** [comment]
Hi @balahari18 @juliantap 

I would like to reword this as follows:

Can Trident provision volumes on SVMs that have a Load-Sharing Mirror configured?
Load-Sharing mirrors can be created for root volumes of SVMs that serve data over NFS. ONTAP automatically updates load-sharing mirrors for volumes that have been created by Trident. This may result in delays in mounting volumes. When multiple volumes are created using Trident, provisioning a volume is dependent on ONTAP updating the load-sharing mirror.

Let me know what you think
